explainaboard/processors/qa_open_domain.py

```diff
@@ -22,72 +28,69 @@ class QAOpenDomainProcessor(Processor):
     def task_type(cls) -> TaskType:
         return TaskType.qa_open_domain
 
-    @classmethod
-    def default_features(cls) -> feature.Features:
-        return feature.Features(
-            {
-                "question": feature.Value("string"),
-                "question_types": feature.Sequence(feature=feature.Value("string")),
-                "answers": feature.Sequence(feature=feature.Value("string")),
-                "question_length": feature.Value(
-                    dtype="float",
-                    description="the length of question",
-                    is_bucket=True,
-                    bucket_info=feature.BucketInfo(
-                        method="bucket_attribute_specified_bucket_value",
-                        number=4,
-                        setting=(),
-                    ),
-                ),
-                "question_type": feature.Value(
-                    dtype="string",
-                    description="question type",
-                    is_bucket=True,
-                    bucket_info=feature.BucketInfo(
-                        method="bucket_attribute_discrete_value", number=4, setting=1
-                    ),
+    def default_analyses(self) -> list[AnalysisLevel]:
+        features = {
+            "question": feature.Value("string"),
+            # "question_types": feature.Sequence(feature=feature.Value("string")),
+            "answers": feature.Sequence(feature=feature.Value("string")),
+            "question_length": feature.Value(
+                dtype="float",
+                description="context length in tokens",
+                func=lambda info, x, c: count_tokens(info, x['question']),
+            ),
+            # "question_type": feature.Value(
```

Comment:
Could you remove commented code? Since we are using CVS, there is almost no reason to keep unused code.